afs/media/js/views/components/workflows/sample-taking-workflow/sample-taking-sample-location-step.js

```diff
@@ -657,7 +660,7 @@ define([
                     cardwidgets: self.card.params.cardwidgets,
                 });
 
-                self.selectedSampleLocationInstance(self.builtTile);
+                self.selectSampleLocationInstance(self.builtTile);
```

Comment:
This change looks like it introduces a bug. If you create a sample with a polygon, save, then create another sample with a point, you'll notice that you can't see the point annotation unless you refresh the page. Removing this change, the dirty state issue looks resolved by the changes on lines 104 and 111.

Reply:
interesting... updated.